src/help-widget.spec.ts

```diff
@@ -333,6 +332,20 @@ describe('BBHelpHelpWidget', () => {
     done();
   });
 
+  it('should override its onHelpLoaded variable with one from the config (function value)', (done) => {
+    const fakeConfig = {
+      onHelpLoaded: () => {
+        return {};
+      }
+    };
+
+    helpWidget['loadCalled'] = false;
+    helpWidget.load(fakeConfig);
+
+    expect(typeof helpWidget.onHelpLoaded).toBe(typeof Function);
```

Comment:
Just to be explicit here that it's the function from the config, have it return a specified value, and call it `helpWidget.onHelpLoaded` to be sure the return value is the same.  Makes for a more well rounded test.